96-chromebook.install

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/bash
+
+COMMAND="$1"
+KERNEL_VERSION="$2"
+BOOT_DIR_ABS="$3"
+KERNEL_IMAGE="$4"
+
+if [[ -d "$BOOT_DIR_ABS" ]]; then
+    INITRD="initrd"
+else
+    BOOT_DIR_ABS="/boot"
+    INITRD="initramfs-${KERNEL_VERSION}.img"
+fi
+
+case "$COMMAND" in
+    add)
+        echo "Install required packages"
+        dnf install lz4 vboot-utils uboot-tools -y
+        if [[ -f "$BOOT_DIR_ABS/$INITRD" ]]; then
+            rm -f -- "$BOOT_DIR_ABS/$INITRD"
+        fi
+        dracut --force -v --force-drivers "ulpi usb-storage phy-qcom-usb-hs-28nm \
+        phy-qcom-usb-ss ocmem dwc3 dwc3-of-simple dwc3-pci ehci-platform xhci-plat-hcd \
+        i2c-qcom-geni i2c-qup icc-osm-l3 qcom-spmi-pmic phy-qcom-qmp phy-qcom-qusb2 \
+        phy-qcom-usb-hs qcom_aoss qcom-apcs-ipc-mailbox llcc-qcom nvmem_qfprom smem \
+        smp2p dwc3-qcom \
+        " "$BOOT_DIR_ABS/$INITRD" --kver "$KERNEL_VERSION" --kmoddir /lib/modules/"${KERNEL_VERSION}"
```

Comment:
I don't think all this has to be done here? this be handled by the dracut kernel-install plugin (`/usr/lib/kernel/install.d/50-dracut.install`) already.

Reply:
this was the list of required modules provided by enric for the initramfs, so this section will regenerate an initramfs to suit the chromebook.

Reply:
yes, that makes sense for creating an initial initrd but that's not needed when updating a kernel package. Since dracut would run on the target machine in that case and would figure out what modules need to be included.

Reply:
Okay, I see. Thanks